roles/cockpit/tasks/subtasks/vm.yml

```diff
@@ -0,0 +1,31 @@
+#########################################################################
+# Title:         Sandbox: Cockpit | VM Tasks                            #
+# Author(s):     CHAIR/Raneydazed                                       #
+# URL:           https://github.com/saltyorg/Sandbox                    #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: VM | Gather package facts
+  ansible.builtin.package_facts:
+
+- name: VM | Install necessary packages for Cockpit Machines
+  ansible.builtin.package:
+    name:
+      - cockpit-machines
```

Comment:
shouldn't this be from the same backport repo as cockpit?

Reply:
Yes, you're right. I overlooked that. My bad